spec/presenters/video_presenter_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'spec_helper'
+
+describe VideoPresenter do
+  
+  describe :display_large_video_image do
+    
+    it "should display a large video image if one is available" do
```

Comment:
I've been trying to get out of that habit but every so often I forget and end up dropping it in again!